src/ng/animate.js

```diff
@@ -233,6 +234,45 @@ var $AnimateProvider = ['$provide', /** @this */ function($provide) {
 
   /**
    * @ngdoc method
+   * @name $animateProvider#customFilter
+   *
+   * @description
```

Comment:
I would add two things:
- How does this relate to the classFilter, .i.e. what takes precedence, or do both run
- Note that this runs for every `animatable` action, not just for animations that are actually defined, and that the user should keep the function fast

You've added this to the guide, but I think it should be here too